app/views/local_authorities/_filter.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="app-view-filter govuk-!-margin-right-5 govuk-!-padding-5">
+  <%= form_with url: local_authorities_path, method: :get do |form| %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Filter by",
+      margin_bottom: 4,
+    } %>
+
+    <%= render "govuk_publishing_components/components/checkboxes", {
+      name: "show_retired",
+      small: true,
+      items: [
+        {
+          label: "Include retired councils",
+          value: "1",
+          bold: true,
+          checked: params["show_retired"],
+        },
+      ],
+    } %>
+
+    <%= render "govuk_publishing_components/components/checkboxes", {
+      name: "only_homepage_problems",
+      small: true,
+      items: [
+        {
+          label: "Only with homepage problems",
+          value: "1",
+          bold: true,
+          checked: params["only_homepage_problems"],
+        },
+      ],
+    } %>
+
```

Comment:
I think we might have a ready pattern we can use here in the option select component https://components.publishing.service.gov.uk/component-guide/option_select.

Reply:
Using this now.